src/pages/formats.mdx

```diff
@@ -0,0 +1,80 @@
+import { Page, Section } from '@layouts'
+
+export default Page
+
+export const title = 'Session Formats'
+
+export const meta = {
+  authors: ['foo', 'bar'],
+  layout: Page,
+  className: 'mw7',
+}
+
+# What will I learn at IPFS Camp?
+
+### Session Types
+
+- 👩🏽‍🏫 [Core & Elective Courses](#core)
+- 📃 [Poster Sessions](#poster)
+- 🐋 [Deep Dives](#deepdive)
+- ⚡️ [Lighting Talks](#lighting)
+- 🧩 [Unconf](#unconf)
+- 🧬 [Sci-Fi Fair](#scifi)
+
+<Section id="core">
+
+## 👩🏽‍🏫 Core & Elective Courses
+
+The Courses of the InterPlanetary Space Training Program (aka IPFS Camp) are one of the main sessions for the event. These are designed to train new IPNauts and the more seasoned IPBuilders in all the arts of the DWeb and P2P.
+
+The Courses are intensive workshops of 75 minutes each. The Core Courses will be a series of sessions to level up their understanding so that we all share a great baseline of understanding, while the Elective Courses will be on even more bleeding edge topics of the DWeb, where each participant decides which ones to attend to advance their knowledge of understanding.
```

Comment:
```suggestion
The Courses are intensive workshops of 75 minutes each. The Core Courses will be a series of sessions to level-up each participant's understanding so that we all share a great baseline to build upon. The Elective Courses will cover even more bleeding-edge topics of the DWeb, where each participant decides which ones to attend to advance their knowledge and deepen their understanding.
```